pkg/socket/socket.go

```diff
@@ -5,10 +5,15 @@ import (
 	"log"
 
 	"dish/pkg/config"
-
-	//"go.savla.dev/swapi/dish"
 )
 
+type Result struct {
+	Socket       Socket
+	Passed       bool
+	ResponseCode int
+	Error        error
+}
+
 type SocketList struct {
 	Sockets map[string]Socket `json:"items"`
 	//Sockets map[string]dish.Socket `json:"items"`
```

Comment:
Do not litter the codebase with old, commented out code. If you need to check something in previous implementations then refer to the commit history.